lab2/cleaner.py

```diff
@@ -0,0 +1,23 @@
+import logging
+import shutil
+from datetime import datetime
+
+from config import LOG_LEVEL, MODEL_DIR, TEST_DIR, TRAIN_DIR
+
+logging.basicConfig()
+logger = logging.getLogger(__name__)
+logger.setLevel(LOG_LEVEL)
+
+
+def clean() -> None:
+    shutil.rmtree(TRAIN_DIR)
+    shutil.rmtree(TEST_DIR)
+    shutil.rmtree(MODEL_DIR)
+

```

Comment:
Можно сделать более лаконично:
```suggestion

def clean_directories():
    directories = [TRAIN_DIR, TEST_DIR, MODEL_DIR]
    for directory in directories:
        shutil.rmtree(directory)

def clean():
    clean_directories()

```

Reply:
3 строки против 8 -- разве это лаконичность?